src/Cooked/MockChain/MockChainSt.hs

```diff
@@ -222,15 +235,13 @@ utxoIndex0From (InitialDistribution initDist) = case mkBody of
   where
     mkBody :: Either GenerateTxError (Cardano.TxBody Cardano.ConwayEra)
     mkBody = do
-      value <- mapLeft (ToCardanoError "Value error") $ Ledger.toCardanoValue (foldl' (\v -> (v <>) . view txSkelOutValueL) mempty initDist)
-      let mintValue = flip (Cardano.TxMintValue Cardano.MaryEraOnwardsConway) (Cardano.BuildTxWith mempty) . Cardano.filterValue (/= Cardano.AdaAssetId) $ value
```

Comment:
Out of curiosity, why was it there in the first place?

Reply:
I have no idea. 
I assumed it had a purpose but I could not find any, and removing it does not change the library's behavior so I'm assuming it was an oversight.